lib/web/jquery/jquery.cookie.js

```diff
@@ -47,7 +47,7 @@
                 options.path    ? '; path=' + options.path : '',
                 options.domain  ? '; domain=' + options.domain : '',
                 options.secure  ? '; secure' : '',
-                options.samesite  ? '; samesite=' + options.samesite : 'lax',
+                options.samesite  ? '; samesite=' + options.samesite : '',
```

Comment:
Feels like it will work not correctly.
THe default value for `samesite` should be `lax`. So I would suggest to change it to something like this:
```suggestion
                '; samesite=' + (options.samesite  ? options.samesite : 'lax'),
```